userspace/engine/rule_loader_reader.cpp

```diff
@@ -807,7 +810,7 @@ bool rule_loader::reader::read(rule_loader::configuration& cfg, collector& colle
 		cfg.res->add_error(falco::load_result::LOAD_ERR_YAML_PARSE, "unknown YAML parsing error", ctx);
 		return false;
 	}
-
+	cfg.res->set_schema_validation_status(schema_validation);
```

Comment:
We will have a schema validation status only if the YAML could be actually loaded, otherwise it will have its default value "none". Not a big deal since an error is thrown if we cannot parse the YAML of course.